light-clients/arbitrum-light-client/types/src/header.rs

```diff
@@ -31,7 +31,7 @@ pub mod proto {
     impl_proto_via_try_from_into!(Header => protos::union::ibc::lightclients::arbitrum::v1::Header);
 
     impl TryFrom<protos::union::ibc::lightclients::arbitrum::v1::Header> for Header {
-        type Error = TryFromHeaderError;
```

Comment:
just because you like this name better? any other reason?

Reply:
Consistency with all other types. Intended usage is `<module name>::proto::Error`, it looks a lot cleaner as well and makes it easier to maintain a naming scheme